### PR TITLE
CODENVY-401: move a product info provider to plugin-product-info

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@codenvy.com>

### CODE REVIEW

File: plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.plugin.product.info.client;
+
+import com.google.gwt.i18n.client.Messages;
+
+
+/**
+ * Che product information constant.
```
Comment (sleshchenko) [inline plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.java]: Codenvy?

File: plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * @author Oleksii Orel
+ */
+public interface CodenvyLocalizationConstant extends Messages {
+
+    @Key("che.tab.title")
```
Comment (sleshchenko) [inline plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.java]: codenvy?

File: plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyProductInfoDataProvider.java [hunk 3]
```diff
@@ -12,49 +12,51 @@
...
  * @author Alexander Andrienko
+ * @author Oleksii Orel
  */
+@Singleton
 public class CodenvyProductInfoDataProvider implements ProductInfoDataProvider {
-
-    private final HostedLocalizationConstant hostedLocalizationConstant;
-    private final HostedResources            resources;
+    private final CodenvyLocalizationConstant locale;
+    private final CodenvyResources             resources;
```
Comment (sleshchenko) [inline plugins/plugin-product-info/src/main/java/com/codenvy/plugin/product/info/client/CodenvyProductInfoDataProvider.java]: formatting

File: plugins/plugin-product-info/src/main/resources/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.properties [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+# and its suppliers and may be covered by U.S. and Foreign Patents,
+# patents in process, and are protected by trade secret or copyright law.
+# Dissemination of this information or reproduction of this material
+# is strictly forbidden unless prior written permission is obtained
+# from Codenvy S.A..
+#
+
+che.tab.title = Eclipse Che
+che.tab.title.with.workspace.name = Eclipse Che | {0}
+get.support.link = https://eclipse.org/che
```
Comment (sleshchenko) [inline plugins/plugin-product-info/src/main/resources/com/codenvy/plugin/product/info/client/CodenvyLocalizationConstant.properties]: Are you sure that it is correct link for codenvy packaging?

**Comment by sleshchenko** [comment]
OK for me

**Comment by ashumilova** [comment]
+1

**Pull request merged**